docs/usage/configuration-options.md

```diff
@@ -1514,7 +1514,8 @@ For example you have multiple `package.json` and want to use `dependencyDashboar
 ```
 
 Important to know: Renovate will evaluate all `packageRules` and not stop once it gets a first match.
-You should order your `packageRules` in order of importance so that later rules can override settings from earlier rules if needed.
+You should order your `packageRules` in order of ascending importance so that more important rules come later
+and can override settings from earlier rules if needed.
```

Comment:
Please combine to be one line

Reply:
Done - I've updated this to be one line. Thanks!